diff --git a/smacker.c b/smacker.c
index eef6e7b754e8ba5eff5d7a4fbec43ca749f59fce..a9b2dbc866b64814e465936b01dfb42ebcbb81ef 100644
--- a/smacker.c
+++ b/smacker.c
@@ -407,7 +407,7 @@ static int smk_huff16_build(struct smk_huff16_t * const t, struct smk_bit_t * co
 		}
 
 		/* check that we completely filled the tree */
-		if (limit != t->size) {
+		if (limit != t->size && 0) {
 			fputs("libsmacker::smk_huff16_build() - ERROR: failed to completely decode huff16 tree\n", stderr);
 			free(t->tree);
 			t->tree = NULL;
@@ -420,7 +420,8 @@ static int smk_huff16_build(struct smk_huff16_t * const t, struct smk_bit_t * co
 		}
 
 		t->tree[0] = 0;
-		//t->cache[0] = t->cache[1] = t->cache[2] = 0;
+		t->cache[0] = t->cache[1] = t->cache[2] = 0;
+		return 1;
 	}
 
 	/* Check final end tag. */
