diff --git forkSrcPrefix/examples/CMakeLists.txt forkDstPrefix/examples/CMakeLists.txt
index d59218c359c1333a5e245f5ab34eee8c8316ef07..4208b62964599ee8eb62818d8272723693b4f099 100644
--- forkSrcPrefix/examples/CMakeLists.txt
+++ forkDstPrefix/examples/CMakeLists.txt
@@ -1,8 +1,14 @@
 
 if(USE_SYSTEM_AUDIO_LIBRARIES OR USE_SYSTEM_SDL2)
     find_package(SDL2 REQUIRED)
+    set(SDL2_LIBRARIES "")
     if(TARGET SDL2::SDL2)
-        set(SDL2_LIBRARIES SDL2::SDL2 SDL2::SDL2main)
+        list(APPEND SDL2_LIBRARIES SDL2::SDL2)
+    elseif(TARGET SDL2::SDL2-static)
+        list(APPEND SDL2_LIBRARIES SDL2::SDL2-static)
+    endif()
+    if(TARGET SDL2::SDL2main)
+        list(APPEND SDL2_LIBRARIES SDL2::SDL2main)
     endif()
     if(WIN32)
         find_library(SDL2main_LIBRARY NAMES SDL2main${MIX_DEBUG_SUFFIX} SDL2main)
diff --git forkSrcPrefix/cmake/libSDL2.cmake forkDstPrefix/cmake/libSDL2.cmake
index 617f6f8014d989e1f153f22bb930b4a22a438528..eff6c3617240edeb56cb7238bf5682818c57886b 100644
--- forkSrcPrefix/cmake/libSDL2.cmake
+++ forkDstPrefix/cmake/libSDL2.cmake
@@ -33,10 +33,14 @@ if(USE_SYSTEM_AUDIO_LIBRARIES OR USE_SYSTEM_SDL2)
         set(CMAKE_FIND_ROOT_PATH "${CMAKE_FIND_ROOT_PATH_OLD2}")
     else()
         find_package(SDL2 REQUIRED)
+        set(SDL2_LIBRARIES "")
         if(TARGET SDL2::SDL2)
-            set(SDL2_LIBRARIES SDL2::SDL2main SDL2::SDL2)
+            list(APPEND SDL2_LIBRARIES SDL2::SDL2)
         elseif(TARGET SDL2::SDL2-static)
-            set(SDL2_LIBRARIES SDL2::SDL2main SDL2::SDL2-static)
+            list(APPEND SDL2_LIBRARIES SDL2::SDL2-static)
+        endif()
+        if(TARGET SDL2::SDL2main)
+            list(APPEND SDL2_LIBRARIES SDL2::SDL2main)
         endif()
     endif()
     set(LIBSDL2CUSTOM_LIB ${SDL2_LIBRARIES})
