diff --git forkSrcPrefix/CMakeLists.txt forkDstPrefix/CMakeLists.txt
index 6c1a5e9770327a7c6068b8253534587d7669e4e3..8cbb0e3e38db6e9e3d6f7d18cedf1fcb265aafa9 100644
--- forkSrcPrefix/CMakeLists.txt
+++ forkDstPrefix/CMakeLists.txt
@@ -7,6 +7,8 @@ set(SDLMixerX_VERSION_PATCH 0)
 set(SDLMixerX_VERSION_TWEAK 0)
 set(SDLMixerX_VERSION      "${SDLMixerX_VERSION_MAJOR}.${SDLMixerX_VERSION_MINOR}.${SDLMixerX_VERSION_PATCH}.${SDLMixerX_VERSION_TWEAK}")
 
+set(SDL_REQUIRED_VERSION 2.0.9)
+
 # =================Library ABI version=======================
 
 # Calculate a libtool-like version number
@@ -557,6 +559,7 @@ install(FILES
 )
 
 if(NOT ANDROID)
+    set(SDL_VERSION ${SDL_REQUIRED_VERSION})
     set(prefix ${CMAKE_INSTALL_PREFIX})
     set(exec_prefix "\${prefix}")
     set(libdir "\${exec_prefix}/lib${LIB_SUFFIX}")
